### PR TITLE
CIV-6000 - CCD- Judgment amount summary screen implementation for CCJ journey 

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-6000

### Change description ###
ccd changes for judgment amount summary.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/staffLRspec-CUI-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    "UserRole": "caseworker-civil-staff",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "ccjPaymentPaidSomeAmount",
+    "UserRole": "caseworker-civil-staff",
+    "CRUD": "R"
+  },
+  {
```
Comment (neeta-hmcts) [inline ccd-definition/AuthorisationCaseField/staffLRspec-CUI-nonprod.json:86]: CIV_6000 changes

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+          "caseworker-civil-admin",
+          "RES-SOL-ONE-UNSPEC-PROFILE",
+          "RES-SOL-ONE-SPEC-PROFILE",
+          "RES-SOL-TWO-SPEC-PROFILE"
+        ],
+        "CRUD": "R"
+      }
+    ]
+  },
+  {
```
Comment (neeta-hmcts) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec-CUI-nonprod.json:301]: CIV_6000 changes

File: ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "ccjPaymentPaidSomeAmount",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeID": "CIVIL",
```
Comment (neeta-hmcts) [inline ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json:87]: CIV_6000 changes

File: ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json [hunk 4]
```diff
@@ -0,0 +1,548 @@
...
+    "CaseFieldID": "ccjPaymentPaidSomeAmount",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "MANDATORY",
+    "PageID": "CcjPaymentPaidSome",
+    "ShowSummaryChangeOption": "Y",
+    "FieldShowCondition": "ccjPaymentPaidSomeOption =\"Yes\"",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/mid/validate-amount-paid"
+  },
+  {
```
Comment (neeta-hmcts) [inline ccd-definition/CaseEventToFields/ClaimantResponseLRspec-CUI-nonprod.json:353]: CIV_6000 changes

File: ccd-definition/CaseField/CaseFieldLRspec-CUI-nonprod.json [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "ccjPaymentPaidSomeAmount",
+    "Label": "Amount already paid",
+    "FieldType": "MoneyGBP",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
```
Comment (neeta-hmcts) [inline ccd-definition/CaseField/CaseFieldLRspec-CUI-nonprod.json:109]: CIV_6000 changes

**Review by laperla11 — APPROVED** [review]

File: e2e/fixtures/events/claimantResponseSpecCui.js [hunk 6]
```diff
@@ -0,0 +1,132 @@
+const {listElement} = require('../../api/dataHelper');
```
Comment (lfazelyniaHMCTS) [inline e2e/fixtures/events/claimantResponseSpecCui.js:1]: Please add your tests in this file for the new fields you've created

**Review by lfazelyniaHMCTS — CHANGES_REQUESTED** [review]

File: e2e/fixtures/events/claimantResponseSpecCui.js (hunk 6 above)
Comment (neeta-hmcts) [reply e2e/fixtures/events/claimantResponseSpecCui.js:1]: done.

File: ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json (hunk 3 above)
Comment (sampankumar) [reply ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json:87]: Are all these fields getting updated as part of Camunda process?
I believe caseworker-civil-systemupdate is used for fields which get updated by any Camunda process

Comment (neeta-hmcts) [reply ccd-definition/AuthorisationCaseField/systemUpdateLRspec-CUI-nonprod.json]: fixed.

**Review by lfazelyniaHMCTS — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

**Review by lfazelyniaHMCTS — APPROVED** [review]

**Pull request closed without merging**